code/datums/json_savefile.dm

```diff
@@ -41,15 +41,13 @@
 	if(!fexists(path))
 		return FALSE
 	try
-		tree = json_decode(file2text(path))
+		tree = json_decode(rustg_file_read(path))
```

Comment:
This doesn't need to be rust-g, we already macro this anyway.

The only reason I asked for rustg_file_write on the other side is it's the only way to write to a file in BYOND without deleting/appending.